cluster/config-defaults.yaml

```diff
@@ -1252,14 +1252,21 @@ role_sync_controller_enabled: "false"
 # When wiz_enable_runtime_sensor and wiz_enable_runtime_connector are set to true,
 # this enables WIZ runtime monitoring. A DaemonSet called Sensor and a Deployment
 # called Connector will be deployed into the cluster.
-wiz_enable_runtime_sensor: "false"
+{{ if eq .Cluster.Provider "zalando-eks" }}
 wiz_enable_runtime_connector: "false"
 wiz_enable_runtime_connector_broker: "false"
+{{ else }}
+wiz_enable_runtime_connector: "true"
+wiz_enable_runtime_connector_broker: "true"
```

Comment:
e2e is breaking since this change. Before wiz was not enabled in e2e?

@hagenmathis